README.md

```diff
@@ -378,6 +378,9 @@ AAAI Conf. Artificial Intelligence, 2020. [PDF](https://www.aaai.org/Papers/AAAI
 - <a name="Monforte20aicas"></a>Monforte, M., Arriandiaga, A., Glover, A., Bartolozzi, C.,  
 *[Exploiting Event Cameras for Spatio-Temporal Prediction of Fast-Changing Trajectories](https://arxiv.org/pdf/2001.01248)*,  
 IEEE Int. Conf. Artificial Intelligence Circuits and Systems (AICAS), 2020.
+- <a name="Sengupta2021aicas"></a>Sengupta, J. P., Kubendran, R., Neftci, E., Andreou, A. G.,  
```

Comment:
Thanks for the updates.
Note that in the keys we only use the last 2 digits of the year 2021 -> 21, so Sengupta21aicas.